python/cudf_polars/cudf_polars/dsl/ir.py

```diff
@@ -967,46 +1129,35 @@ def evaluate(self, *, cache: MutableMapping[int, DataFrame]) -> DataFrame:
         return result.slice(self.zlice)
 
 
-@dataclasses.dataclass
 class Sort(IR):
     """Sort a dataframe."""
 
-    df: IR
-    """Input."""
-    by: list[expr.NamedExpr]
-    """List of expressions to produce sort keys."""
```

Comment:
I't would be nice to preserve the description of the constructor arguments.

Reply:
Now that everything has a constructor I can move them there.

Reply:
Ah, that would be very long, so I just reinstated.